### DISCUSSION TITLE
Is it possible to show package name in path

### DISCUSSION THREAD

**Discussion by aahnik** [Textualize/rich#1286] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/66209958/121684691-a7b1df00-cadc-11eb-9cd8-796e7b30d51e.png)

The above picture is a screenshot of logs produced by my application [tgcf](https://github.com/aahnik/tgcf). 

I am using the rich handler like this.

```python
    logging.basicConfig(
        level=level,
        format="%(message)s",
        handlers=[
            RichHandler(
                rich_tracebacks=True,
                markup=True,
            )
        ],
    )
```

In the right column you can see the path of file with line no, which emmited the log.

how can I also show the package name ?  (full path like `package.subpackage.module:line`)

how to customize the path format ?

**Comment by willmcgugan** [comment, marked as answer]
There's no way to configure that at the moment, but the RichHandler class could be subclasses to add this. You probably want to customize the [RichHandler.render](https://github.com/willmcgugan/rich/blob/master/rich/logging.py#L174) function.
